src/components/Ticketmaster.js

```diff
@@ -195,6 +218,16 @@ const Ticketmaster = function() {
             </li>
             ))}
     </ResultListMobile>
+    <Pagination role="navigation" aria-label="Pagination">
+      <ul>
+        <li>
+          <button onClick={prevPage} disabled={page === 0} aria-label="Next Page">&laquo;</button>
```

Comment:
Sorry for not being clear but this should be one button which will expand the page vertically with the results.